keras/backend/tensorflow_backend.py

```diff
@@ -280,6 +280,31 @@ def to_dense(tensor):
 name_scope = tf.name_scope
 
 
+def is_placeholder(tensor):
+    """Returns whether a tensor is a placeholder.
+
+    # Arguments
+        tensor: A tensor instance.
+
+    # Returns
+        A boolean.
+
+    # Example
+    ```python
+        >>> from keras import backend as K
+        >>> a = K.placeholder((2, 2), sparse=False)
+        >>> print(K.is_placeholder(a))
+        True
+    ```
+    """
+    try:
+        if isinstance(tensor, tf.Variable):
+            return True
+        return tensor._is_placeholder
+    except AttributeError:
+        return True
```

Comment:
This is for backwards compatibility with layers that don't specify `_is_placeholder`. It should be accompanied by a warning with instructions to update your layers.

1. Should `_is_placeholder` become `is_placeholder`?
2. Should `_is_placeholder` be added to `get_config` of `Container` or some other location?